src/Microsoft.Health.Fhir.Core/Features/Operations/Import/RebuildIndexProgress.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.Import
+{
+    public enum RebuildIndexProgress
+    {
+        Initialized,
+        DisableIndexCompleted,
+        InitialIndexProperties,
```

Comment:
>InitialIndexProperties [](http://example.com/codeflow?start=8&length=22)

Please clarify what state this designates.

Reply:
Ok, I will modify and clarify the definition of them in next commit.